packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -218,9 +218,7 @@ class Splitter extends CollectionWidget<Properties> {
     this._shouldRecalculateLayout = false;
   }
 
-  _renderItems(items: Item[]): void {
-    super._renderItems(items);
-
+  updateItemsLayout(): void {
```

Comment:
This seems very unreliable, abandoning our usual 'renderItems' method and using 'contentReady' to finish what we didn't complete earlier. We can adjust or redraw on 'contentReady,' but we shouldn't move the main logic there—it might not be called, and then what happens? We need to find a better solution.